src/ensembl/src/shared/accordion/components/Accordion.test.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+import { mount } from 'enzyme';
+import Accordion from './Accordion';
+import AccordionItem from './AccordionItem';
+import AccordionItemHeading from './AccordionItem';
+import AccordionItemButton from './AccordionItem';
+
+enum UUIDS {
+  FOO = 'FOO',
+  BAR = 'Bar'
+}
+
+describe('Accordion', () => {
+  it('renders without erroring', () => {
+    expect(() => {
+      mount(<Accordion />);
+    }).not.toThrow();
+  });
+
+  describe('className', () => {
+    it('is “accordion” by default', () => {
+      const wrapper = mount(<Accordion data-testid={UUIDS.FOO} />);
+      expect(wrapper.find('div').props().className).toEqual('accordion');
+    });
+
+    it('can be overridden', () => {
+      const wrapper = mount(
+        <Accordion className="foo" data-testid={UUIDS.FOO} />
+      );
+      expect(wrapper.find('div').props().className).toEqual('foo');
+    });
```

Comment:
Do we want to overwrite the className entirely, or to extend it with a classname received from the parent (so that a portion of styles could be reused)?

compare:  `expect(wrapper.find('div').props().className).toEqual('foo')` vs `expect(wrapper.find('div').props().className).toEqual('accordion foo')` (or `.toContain('foo')`?

Reply:
Ideally I would like to extend it with the classname received, but it will make it hard to update the default styling later on as any changes we make will be applied to all the accordions.